website/content/api-docs/secret/kv/kv-v2.mdx

```diff
@@ -146,11 +148,10 @@ have an ACL policy granting the `update` capability.
 
 - `options` `(Map: <optional>)` – An object that holds option settings.
 
-  - `cas` `(int: <optional>)` - Set the "cas" value to use a Check-And-Set
-    operation. If not set the write will be allowed. If set to 0 a write will
-    only be allowed if the key doesn’t exist. If the index is non-zero the
-    write will only be allowed if the key’s current version matches the
-    version specified in the cas parameter.
+- `cas` `(int: <optional>)` - This flag is required if cas_required is set
+  to true on either the secret or the engine's config. In order for a write
+  to be successful, cas must be set to the current version of the secret.
+  If cas is set to 0, the write will only be allowed if the key doesn't exist.
 
```

Comment:
I started a new line based on what was there before.  However, I wasn't sure if there was a specific max character count or something I should be following for the length of line descriptions?